pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +414,265 @@ def image_threshold(
+        component containing real values and the second component containing
+        imaginary values. The filter is fastest for images that have power of
+        two sizes.
+
+        The filter uses a butterfly diagram for each prime factor of the
+        dimension. This makes images with prime number dimensions (i.e. 17x17)
+        much slower to compute. Multi dimensional (i.e volumes) FFT's are
+        decomposed so that each axis executes serially.
+
+        Parameters
+        ----------
+        output_scalars_name : str, optional
+            The name of the output scalars. By default this is
+            ``'ImageScalars'``.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            UniformGrid with the applied reverse FFT.
+
+        Examples
+        --------
+        Apply reverse FFT to an example image.
+
+        >>> from pyvista import examples
+        >>> image = examples.download_moonlanding_image()
+        >>> fft_image = image.fft()
+        >>> image_again = fft_image.rfft()
+        >>> image_again.point_data  # doctest:+SKIP
+        pyvista DataSetAttributes
+        Association     : POINT
+        Active Scalars  : PNGImage
+        Active Vectors  : None
+        Active Texture  : None
+        Active Normals  : None
+        Contains arrays :
+        PNGImage                float64  (298620, 2)          SCALARS
+
+        See :ref:`image_fft_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageRFFT()
+        alg.SetInputDataObject(self)
+        _update_alg(alg, progress_bar, 'Performing Reverse Fast Fourier Transform.')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def low_pass(
+        self,
+        x_cutoff,
+        y_cutoff,
+        z_cutoff,
+        order=1,
+        output_scalars_name='ImageScalars',
+        progress_bar=False,
+    ):
+        """Perform a low pass filter in the frequency domain.
+
+        This filter only works on an image after it has been converted to
+        frequency domain by a :func:`UniformGridFilters.fft` filter.
+
+        A :func:`UniformGridFilters.rfft` filter can be
+        used to convert the output back into the spatial
+        domain. This filter attenuates high frequency components.
+        Input and output are in doubles, with two components.
+
+        Parameters
+        ----------
+        x_cutoff : double
+            The cutoff frequency for the x axis.
+
+        y_cutoff : double
+            The cutoff frequency for the y axis.
+
+        z_cutoff : double
+            The cutoff frequency for the z axis.
+
+        order : int, optional
+            The order of the cutoff curve. Given from the equation
+             ``(1 + pow(CutOff/Freq(i, j), 2*Order))``.
```

Comment:
This equation is confusing to me for three reasons.

Firstly, it's probably cleaner in python when written out as
```suggestion
             ``1 + (cutoff/freq(i, j))**(2*order)``.
```

Secondly, it's not clear what `freq(i, j)` is; from [the VTK docs](https://vtk.org/doc/nightly/html/classvtkImageButterworthLowPass.html#details) this is a bit clearer:
> Input and output are in doubles, with two components (complex numbers). out(i, j) = (1 + pow(CutOff/Freq(i,j), 2*Order));

This seems to suggest that `i` and `j` are image coordinates, but then this is surprising because the filter seems to work for 3d "images" as well.

Thirdly, I'm not a signal processing person but [Wikipedia makes it sound like](https://en.wikipedia.org/wiki/Butterworth_filter#Original_paper) a Butterworth low-pass filter's frequency response looks like
```py
    G(w) = (1 + w**(2*order))**(-1/2)
```
and [this section](https://en.wikipedia.org/wiki/Butterworth_filter#Transfer_function) rewrites the square of this function as a function of `w/w_c` with the cutoff frequency `w_c`... so it's not clear to me why there's an overall reciprocal missing and why the `w/w_c` term is mentioned with a reciprocal.